app/controllers/sessions_controller.rb

```diff
@@ -8,7 +8,7 @@ def new
       flash[:notice] = "You are already logged in"
       if user.admin_user?
         redirect_to "/admin"
-      elsif user.merchant_admin?
+      elsif user.merchant_admin? || user.merchant_employee?
```

Comment:
Just to double check, did we change the role name to `merchant_admin` from `merchant_user` ?